docker/Dockerfile.test

```diff
@@ -1,16 +1,22 @@
-FROM ghcr.io/openconext/openconext-basecontainers/php72-apache2:latest
+FROM busybox AS unpack
+WORKDIR /unpack
+COPY output.zip /unpack
+RUN unzip /unpack/output.zip
+
+FROM ghcr.io/openconext/openconext-basecontainers/php82-apache2:latest
+# Set the default workdir
 WORKDIR /var/www/html
-ARG APP_VERSION
-ENV GATEWAY_VERSION=$APP_VERSION
-ADD output.tar /var/www/html/
+COPY --from=unpack /unpack/ /var/www/html/
 # Add the application configuration files
 COPY config/openconext/parameters.yaml.dist config/openconext/parameters.yaml
 COPY config/openconext/samlstepupproviders_parameters.yaml.dist config/openconext/samlstepupproviders_parameters.yaml
 
 # Add the config files for Apache2
 RUN rm -rf /etc/apache2/sites-enabled/*
 COPY ./docker/conf/gateway-apache2.conf /etc/apache2/sites-enabled/gateway.conf
-RUN bin/console --env=test cache:clear
-RUN rm -rf /var/www/html/var/cache/test && chown -R www-data /var/www/html/var
+RUN rm -rf /var/www/html/var/cache/prod &&\
+  mkdir -p /var/www/html/var/cache &&\
+  chown -R www-data /var/www/html/var
 
+EXPOSE 80
```

Comment:
The pr title says: php version bump.

Why is port 80 exposed?
Why change cache:clear into rm cache/prod? 
What's the busybox part?

Maybe discuss? Or is this copy/pasted from another project?